### ISSUE TITLE
Basic Authentication for Image URL

### ISSUE THREAD

**Issue by saadfarooq** [mttkay/ignition#10]
I don't know if you would want to consider this or not but basic authentication capabilities for the RemoteImageLoader could be handy in some cases.

**Comment by mttkay** [comment]
Good point! Do you have a URL to test against, one that requires e.g. a BasicAuth handshake or something?

**Comment by saadfarooq** [comment]
I could give you one but then I'd have to kill you... 

No seriously. I'll have to ask someone cause they're pretty secretive but I did the following on line 88 of RemoteImageLoaderJob.java 

```
   HttpURLConnection connection = (HttpURLConnection) url.openConnection();

   connection.setRequestProperty("Authorization", "Basic "+ "bm9xd3RvcxU6dHgpa*w=");
```

No surprise there. All you need is to have a way to pass this down somehow. I can test it for you until I get approval.
